BioArchLinux/snpeff/lilac.yaml

```diff
@@ -5,7 +5,6 @@ build_prefix: extra-x86_64
 post_build_script: |
   git_add_files('PKGBUILD')
   git_commit()
-  update_aur_repo()

```

Comment:
May I know why you removed this line, by mistake or intended?

Reply:
just as a caution. once the compiled package is tested and found fine we  can add update_aur_repo() back. i know for sure that AUR version is working fine so dont want to cause any issues with it for the time being. 🤞

Reply:
> just as a caution. My intention was once the compiled package is tested and found fine we can add update_aur_repo() back. i know for sure that AUR version is working fine so don't want to cause any issues with it for the time being. crossed_fingers
> 
Good idea, you can test the PKGBUILD following the steps
1.  change to the directory that contains PKGBUILD of this pkg, 
2. then use `bioarchlinux-x86_64-build` to build it. 
3. If it is ok, you can then test whether it works fine. 
4. If it works, please add `update_aur_repo()` back.

BTW, if it failed to be built, it won't be pushed to aur even if `update_aur_repo()` is present.

Reply:
i have compiled and installed the package using updated `PKGBUILD`. So far in my usage, the newly built binary seems to be working fine. so, it should be okay to add `update_aur_repo()` back.